demos/common/cpp/models/src/detection_model_ssd.cpp

```diff
@@ -41,6 +42,12 @@ std::shared_ptr<InternalModelData> ModelSSD::preprocess(const InputData& inputDa
 }
 
 std::unique_ptr<ResultBase> ModelSSD::postprocess(InferenceResult& infResult) {
+        return shoulPostprocessMultipleOutputs ?
+            postprocessMultipleOutputs(infResult) :
+            postprocessSingleOutput(infResult);
```

Comment:
```suggestion
    return outputsNames.size() > 1 ?
        postprocessMultipleOutputs(infResult) :
        postprocessSingleOutput(infResult);
```
`shoulPostprocessMultipleOutputs` isn't needed.